CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Bob versions changelog
 - Exist test on doubled range of keys for integration tests (#764)
 - Used swap and bob virtual memory metrics added to hardware metrics (#771)
 - Added validate_data_checksum_in_index_regen field to pearl config (#607)
+- 'Exists' method through HEAD request in REST API (#691)
+- Support for 'exists' method in bobt (#691)
```

Comment:
Those 2 lines for #691 are duplicate and should be removed. These changes was merged within `v2.1.0-alpha.8`